Update terminology in JWS JSON Serialization (#427)
oauth-wg/oauth-selective-disclosure-jwt: draft-ietf-oauth-selective-disclosure-jwt.md

# JWS JSON Serialization {#json_serialization}

This section describes an alternative format for SD-JWTs and SD-JWT+KBs using the JWS JSON
Serialization from [@!RFC7515]. Supporting this format is OPTIONAL.

## New Unprotected Header Parameters {#json_serialization_unprotected_headers}

For both the General and Flattened JSON Serialization, the SD-JWT or SD-JWT+KB is represented
as a JSON object according to Section 7.2 of [@!RFC7515]. The following new
unprotected header parameters are defined:

 * `disclosures`: An array of strings where each element is an individual
   Disclosure as described in (#creating_disclosures).

>  * `kb_jwt`: Only in case of an SD-JWT+KB, the Key Binding JWT as described in (#kb-jwt).

Reviewer: ```suggestion
 * `kb_jwt`: Present only in SD-JWT+KB, the Key Binding JWT as described in (#kb-jwt).
```

# JWS JSON Serialization {#json_serialization}

This section describes an alternative format for SD-JWTs and SD-JWT+KBs using the JWS JSON
Serialization from [@!RFC7515]. Supporting this format is OPTIONAL.

## New Unprotected Header Parameters {#json_serialization_unprotected_headers}

For both the General and Flattened JSON Serialization, the SD-JWT or SD-JWT+KB is represented
as a JSON object according to Section 7.2 of [@!RFC7515]. The following new
unprotected header parameters are defined:

 * `disclosures`: An array of strings where each element is an individual
   Disclosure as described in (#creating_disclosures).

>  * `kb_jwt`: Present only in SD-JWT+KB, the Key Binding JWT as described in (#kb-jwt).

Reviewer 2: I think you need/want an indefinite article* here. 
```suggestion
 * `kb_jwt`: Present only in an SD-JWT+KB, the Key Binding JWT as described in (#kb-jwt).
```
* yes I had to look that up
